app/matplotlib.ipynb

```diff
@@ -233,22 +307,44 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
-   "outputs": [],
    "source": [
     "x = np.linspace(0, 5, 100)\n",
     "y1 = np.sin(x)\n",
     "y2 = np.cos(x)\n",
     "\n",
-    "# write your code here\n"
+    "fig, (plt1, plt2) = plt.subplots(1, 2, figsize=(12, 4))\n",
+    "fig.suptitle('Sin and cos figures')\n",
+    "\n",
+    "plt1.plot(y1, 'tab:green')\n",
+    "plt1.set_title(\"Sin function\")\n",
+    "plt1.set(xlabel=\"x-axis\", ylabel=\"y-axis\")\n",
+    "\n",
+    "plt2.set_title(\"Cos function\")\n",
+    "plt2.plot(y2, 'tab:red')\n",
```

Comment:
The `plot` function should include the `x` variable to correctly plot the sin and cos functions against the x-axis values. Use `plt1.plot(x, y1, 'tab:green')` and `plt2.plot(x, y2, 'tab:red')`.